tests/integration/test_data/private_repo_packages.yaml

```diff
@@ -8,3 +8,465 @@ private_repo_https:
+            type: go-package
+            version: null
+          - name: os
+            replaces: null
+            type: go-package
+            version: null
+          - name: path
+            replaces: null
+            type: go-package
+            version: null
+          - name: reflect
+            replaces: null
+            type: go-package
+            version: null
+          - name: rsc.io/quote
+            replaces: null
+            type: go-package
+            version: v1.5.2
+          - name: rsc.io/sampler
+            replaces: null
+            type: go-package
+            version: v1.3.0
+          - name: runtime
+            replaces: null
+            type: go-package
+            version: null
+          - name: runtime/internal/atomic
+            replaces: null
+            type: go-package
+            version: null
+          - name: runtime/internal/math
+            replaces: null
+            type: go-package
+            version: null
+          - name: runtime/internal/sys
+            replaces: null
+            type: go-package
+            version: null
+          - name: runtime/internal/syscall
+            replaces: null
+            type: go-package
+            version: null
+          - name: sort
+            replaces: null
+            type: go-package
+            version: null
+          - name: strconv
+            replaces: null
+            type: go-package
+            version: null
+          - name: strings
+            replaces: null
+            type: go-package
+            version: null
+          - name: sync
+            replaces: null
+            type: go-package
+            version: null
+          - name: sync/atomic
+            replaces: null
+            type: go-package
+            version: null
+          - name: syscall
+            replaces: null
+            type: go-package
+            version: null
+          - name: time
+            replaces: null
+            type: go-package
+            version: null
+          - name: unicode
+            replaces: null
+            type: go-package
+            version: null
+          - name: unicode/utf8
+            replaces: null
+            type: go-package
+            version: null
+          - name: unsafe
+            replaces: null
+            type: go-package
+            version: null
+        name: github.com/cachito-testing/cachito-gomod-with-deps-private
+        type: go-package
+        version: v0.0.0-20230214150328-0fc5a4360594
+      - dependencies:
+          - name: golang.org/x/text
+            replaces: null
+            type: gomod
+            version: v0.0.0-20170915032832-14c0d48ead0c
+          - name: rsc.io/quote
+            replaces: null
+            type: gomod
+            version: v1.5.2
+          - name: rsc.io/sampler
+            replaces: null
+            type: gomod
+            version: v1.3.0
+        name: github.com/cachito-testing/cachito-gomod-with-deps-private
```

Comment:
> G.9 With private dependencies

Wasn't the point of the test that one of the dependencies would be a private repo?

Reply:
oh right, the description of the test is pretty vague. I'll fix this.